src/main/java/io/ipinfo/api/model/IPResponse.java

```diff
@@ -95,6 +95,10 @@ public String getCountryName() {
         return context.getCountryName(getCountryCode());
     }
 
+    public boolean isCountryEU() {
```

Comment:
Aren't we calling it just `isEU` in the other SDKs? We should keep the SDKs really consistent if possible.